src/pages/inventory/index.tsx

```diff
@@ -41,7 +41,7 @@ export function Inventory() {
 
   const { addFilter, getActiveFilterItems } = filterContext;
 
-  // First create a lookup table for all aid requests grouped by dates and memoise it
+  // First create a lookup table for all stock items grouped by dates and memoise it
```

Comment:
**issue**

Why this change? `AidRequest` was / is a data model designed to conform to: 
- business requirements
- BE data structure

I have no idea, what a stock item is and where it is coming from.

Besides, an AidRequest can have multiple stock items, I guess.

Reply:
Ok, sorry, it's inventory, not requests. My bad!